tests/main/security-dev-input-event-denied/task.yaml

```diff
@@ -63,3 +73,30 @@ execute: |
     # joystick interface is connected since a keyboard shouldn't be added to
     # the device cgroup.
     MATCH "Operation not permitted" < call.error
+
+    # 2. Device Buttons
+
+    echo "Then the snap is not able to access an evdev keyboard"
+    if test-snapd-event "-event-kbd" 2>"${PWD}"/call.error; then
+        echo "Expected permission error calling evtest with disconnected plug"
+        exit 1
+    fi
+    # AppArmor is 'Permission denied' which is expected with default policy
+    MATCH "Operation not permitted" < call.error
```

Comment:
The comment here doesn't go with the MATCH statement and what this is showing is that you have a device cgroup denial. You should disconnect the joystick interface (with a comment) before ' # 2. Device Buttons', then use:
```
MATCH "Permission denied" < call.error
```

Reply:
I fixed that and ran into issues. With `device-buttons` connected, none of the devices matched, hence `snap-confine` did not put itself under device cgroup in the first place. As a result, the test snap had unconfined access to all input devices. I'll address that in a separate PR.

Reply:
This was why I wanted the spread tests! :) Thanks for finding this. In this PR, please add "/dev/full" to the cgroup like we did in the joystick interface here: https://github.com/snapcore/snapd/pull/5231/files

If/when the unconditional cgroup is added (ie, your other PR), then this /dev/full here and in the joystick interface can be removed.